### ISSUE TITLE
Why log_prob * self.td_error is not log_prob * tf.stop_gradient(self.td_error)?

### ISSUE THREAD

**Issue by exceeddream** [MorvanZhou/Reinforcement-learning-with-tensorflow#90]
In your code of A3C, exp_v = log_prob * tf.stop_gradient(td), here stop the gradient of td. However, in your code of AC, self.exp_v = tf.reduce_mean(log_prob * self.td_error), here not stop the gradient of td_error.

Why they are different?  Could you explain?  Thank you!
